docs/docs/manual/columnediting.md

```diff
@@ -0,0 +1,128 @@
+
+Column editing contains some of the most powerful data-improvement methods in OpenRefine. While we call it “edit column,” in fact many of the best features involve using one column of data to add entirely new columns and fields to your dataset. 
+
+## Split or Join
+
+Many users find that they frequently need to make their data more granular: for example, splitting a “Firstname Lastname” column into two columns, one for first names and one for last names. You may want to split out an address column into columns for street addresses, cities, territories, and postal codes. 
+
+The reverse is also often true: you may have several columns of category values that you want to join into one “category” column. 
+
+### Split into several columns...
+
+![A screenshot of the settings window for splitting columns.](/img/columnsplit.png)
+
+Splitting one column into several columns requires you to identify the character, string lengths, or evaluating expression you want to split on. Just like [splitting multi-valued cells into rows](cellediting#split-multi-valued-cells), splitting cells into multiple columns will remove the separator character or string you indicate. Lengths will discard any information that comes after the specified total length. 
+
+You can also specify a maximum number of new columns to be made: separator characters after this limit will be ignored, and the remaining characters will end up in the last column.
+
+New columns will be named after the original column, with a number: “Location 1,” “Location 2,” etc. You can have the original column removed with this operation, and you can have [data types](exploring#data-types) identified where possible. This function will work best with converting to numbers, and may not work with dates.
+
+### Join columns…
+
+![A screenshot of the settings window for joining columns.](/img/columnjoin.png)
+
+You can join columns by selecting “Edit column” > “Join columns…”. All the columns currently in your dataset will appear in the pop-up window. You can select or un-select all the columns you want to join, and drag columns to put them in the order you want to join them in. You will define a separator character (optional) and define a string to insert into empty cells (nulls). 
+
+The joined data will appear in the column you originally selected, or you can create a new column based on this join and specify a name. You can delete all the columns that were used in this join operation. 
+
+## Add column based on this column
+
+This selection will open up an [expressions](expressions) window where you can transform the data from this column (using `value`) or write a complex expression that takes information from any number of columns or from external reconciliation sources. 
+
+One common use of this function is to join information from two projects together in one place. Just like referring from one table to another table in an entity-relationship database, you need a key that exists in both. 
```

Comment:
`cross()` is a pretty advanced function. I think various string manipulation functions are likely much more commonly used.